Gopkg.toml

```diff
@@ -45,6 +45,8 @@ required = [
   name = "k8s.io/apiserver"
   branch = "release-1.8"
 
+# Use munnerz/code-generator fork that includes the filter changes
 [[constraint]]
   name = "k8s.io/code-generator"
   branch = "release-1.8"
```

Comment:
❓ I was going to ask if this is the correct branch in your forked repo, but it looks like you've applied the changes there too: https://github.com/munnerz/code-generator/tree/release-1.8

Reply:
Yep - otherwise there were more changes to generated files that were unrelated. Thought it best to make this 1:1